pages/myDonations.js

```diff
@@ -16,45 +15,70 @@ export default function MyDonations({
   );
   return (
     <>
-      <h2> Meine Spenden:</h2>
-      <AddDonation createDonation={createDonation} />
-      <StyledCard>
-        {filteredDonations.map((donation) => {
-          return (
-            <DonationListItem
-              donations={donations}
-              setDonations={setDonations}
-              key={donation.id}
-              donation={donation}
-              handleDelete={handleDelete}
-              myUserID={myUserID}
-              handleEditDonation={handleEditDonation}
-            />
-          );
-        })}
-      </StyledCard>
+      <Wrapper>
+        <AddDonation createDonation={createDonation} />
+        <StyledHeadline>
+          <StyledH2> Meine Spenden:</StyledH2>
+        </StyledHeadline>
+        <StyledCard>
+          {filteredDonations.map((donation) => {
+            return (
+              <DonationListItem
+                donations={donations}
+                setDonations={setDonations}
+                key={donation.id}
+                donation={donation}
+                handleDelete={handleDelete}
+                myUserID={myUserID}
+                handleEditDonation={handleEditDonation}
+              />
+            );
+          })}
+        </StyledCard>
+      </Wrapper>
       <StyledNavigation as="a" href={"/allDonations"}>
         Ich möchte retten!
       </StyledNavigation>
     </>
   );
 }
 
+const Wrapper = styled.div`
+  padding: 10px;
+  margin-top: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: top;
+  justify-content: center;
+`;
+
 const StyledNavigation = styled.nav`
   z-index: 1;
   text-align: center;
   justify-content: center;
   position: fixed;
-  background-color: green;
+  background-color: #008000;
   padding: 10px 10px 10px 10px;
   margin: 0;
-  width: 100%;
+  width: 100vw;
   bottom: 0;
+  color: white;
   text-decoration: none;
-  color: black;
 `;
+
 const StyledCard = styled.ul`
   list-style: none;
-  margin: 0px;
   padding: 0px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
```

Comment:
As a list has block items by default these styling seem redundant. I think you used flexbox as you want to set a gap. You can use `grid` for this. I would recommend to remove the margins from your list items and use padding and gap in your list instead.